mne/viz/topomap.py

```diff
@@ -1578,11 +1572,268 @@ def _prepare_topomap(pos, ax):
     """Helper for preparing the topomap."""
     pos_x = pos[:, 0]
     pos_y = pos[:, 1]
-
-    ax.set_xticks([])
-    ax.set_yticks([])
-    ax.set_frame_on(False)
+    _hide_frame(ax)
     if any([not pos_y.any(), not pos_x.any()]):
         raise RuntimeError('No position information found, cannot compute '
                            'geometries for topomap.')
     return pos_x, pos_y
+
+
+def _hide_frame(ax):
+    """Helper to hide axis frame for topomaps."""
+    ax.set_xticks([])
+    ax.set_yticks([])
+    ax.set_frame_on(False)
+
+
+def _init_anim(ax, ax_line, ax_cbar, params, merge_grads):
+    """Initialize animated topomap."""
+    import matplotlib.pyplot as plt
+    from matplotlib import patches
```

Comment:
if you really want to squeeze lines you can do `from matplotlib import pyplot as plt, patches` but not totally necessary